launch/ros_control.launch

```diff
@@ -0,0 +1,31 @@
+<launch>
+  <node pkg="libbarrett_ros" type="controller" name="controller" output="screen">
+
+    <rosparam command="load" subst_value="true"
+              file="$(find herb_launch)/config/ros_control.yaml"/>
+
+    <param name="buses/right_arm/configuration_path" type="str"
+           value="$(find herb_launch)/libbarrett_config/right_arm/default.conf"/>
+
+    <param name="buses/left_arm/configuration_path" type="str"
+           value="$(find herb_launch)/libbarrett_config/left_arm/default.conf"/>
+
+    <rosparam command="load" ns="buses/right_arm/schedule"
+              file="$(find herb_launch)/config/ros_control_schedule.yaml"/>
+
+    <rosparam command="load" ns="buses/left_arm/schedule"
+              file="$(find herb_launch)/config/ros_control_schedule.yaml"/>
+  </node>
+
+  <rosparam command="load"
+            file="$(find herb_launch)/config/ros_control_controllers.yaml"/>
+
+  <!-- TODO timeout? -->
+  <node name="ros_control_manager"
+        pkg="controller_manager"
+        type="spawner"
+        args="--stopped --timeout 120 joint_state_controller force_torque_sensor_conroller right_hand_controller right_tare_controller left_hand_controller left_tare_controller right_gravity_compensation_controller left_gravity_compensation_controller right_joint_group_position_controller left_joint_group_position_controller right_trajectory_controller left_trajectory_controller bimanual_trajectory_controller"
```

Comment:
Basically it needs to be long enough to reset, home, and activate the arms. I can't think of a better way to do this other than a shell script I call manually to do the loading after activating the arms.